doc/modules/linear_model.rst

```diff
@@ -175,13 +175,12 @@ This method has the same order of complexity as
 .. between these
 
 
-Setting the regularization parameter: generalized Cross-Validation
+Setting the regularization parameter: leave-one-out Cross-Validation
 ------------------------------------------------------------------
 
 :class:`RidgeCV` implements ridge regression with built-in
 cross-validation of the alpha parameter. The object works in the same way
-as GridSearchCV except that it defaults to Generalized Cross-Validation
-(GCV), an efficient form of leave-one-out cross-validation::
+as GridSearchCV except that it defaults to Leave-One-Out cross-validation::
```

Comment:
It would be great to be consistent in capitalization.
(`leave-one-out Cross-Validation` above vs `Leave-One-Out cross-validation` here)

Reply:
Resolved